mods/ra/maps/allies-06a/map.yaml

```diff
@@ -0,0 +1,650 @@
+MapFormat: 11
+
+RequiresMod: ra
+
+Title: 06a: Iron Curtain research center
+# Infiltrate and destroy Iron Curtain research center
```

Comment:
This needs to be resolved (the original title, commented out here, is too long for the UI).